theano/compile/profilemode.py

```diff
@@ -695,6 +695,29 @@ def exp_float32_op(op):
         if not printed_tip:
             print("  Sorry, no tip for today.")
 
+    def clone(self, link_kwargs=None, message=None):
```

Comment:
This part was just refactoring of existing code, so I would not have written new tests for it anyway, since it does not break existing ones.
We can remove ProfileMode, but I think we can wait a bit for that.